src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -21933,79 +24396,113 @@ static constexpr const rct_vehicle_info_list *TrackVehicleInfoListChairliftStart
+    &RightLane::TrackVehicleInfoLeftSBend0, &RightLane::TrackVehicleInfoLeftSBend1, &RightLane::TrackVehicleInfoLeftSBend2, &RightLane::TrackVehicleInfoLeftSBend3, // ELEM_S_BEND_LEFT
+    &RightLane::TrackVehicleInfoRightSBend0, &RightLane::TrackVehicleInfoRightSBend1, &RightLane::TrackVehicleInfoRightSBend2, &RightLane::TrackVehicleInfoRightSBend3, // ELEM_S_BEND_RIGHT
+    &TrackVehicleInfo_8D44BE, &TrackVehicleInfo_8D4FA6, &TrackVehicleInfo_8D5A8E, &TrackVehicleInfo_8D6576, // ELEM_LEFT_VERTICAL_LOOP
+    &TrackVehicleInfo_8D705E, &TrackVehicleInfo_8D7B46, &TrackVehicleInfo_8D862E, &TrackVehicleInfo_8D9116, // ELEM_RIGHT_VERTICAL_LOOP
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // ELEM_LEFT_QUARTER_TURN_3_TILES
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // ELEM_RIGHT_QUARTER_TURN_3_TILES
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // ELEM_LEFT_QUARTER_TURN_3_TILES_BANK
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // ELEM_RIGHT_QUARTER_TURN_3_TILES_BANK
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp3, // ELEM_LEFT_QUARTER_TURN_3_TILES_25_DEG_UP
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp3, // ELEM_RIGHT_QUARTER_TURN_3_TILES_25_DEG_UP
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown3, // ELEM_LEFT_QUARTER_TURN_3_TILES_25_DEG_DOWN
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown3, // ELEM_RIGHT_QUARTER_TURN_3_TILES_25_DEG_DOWN
```

Comment:
Could you use the full name (i.e. with TRACK_ prefix)?

I see that we did it incorrectly somewhere else too, which is probably why you did it as well.

Reply:
Not quite sure what you mean by this - what specifically are you referring to? The comment?

Reply:
Yeah. The comments now start with `ELEM_`, while the enum members start with `TRACK_ELEM_`.